packages/web/hooks/user-settings/use-user-favorite-asset-denoms.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback } from "react";
+
+import { useLocalStorageState } from "../window/use-localstorage-state";
+
+export function useUserFavoriteAssetDenoms() {
+  const [favoritesList, onSetFavoritesList] = useLocalStorageState(
+    "favoritesList",
+    ["OSMO", "ATOM", "TIA"]
+  );
+
+  const addFavoriteDenom = useCallback(
```

Comment:
Since this is an event callback based on an action by the user, what are your thoughts on appending `on`?

```suggestion
  const onAddFavoriteDenom = useCallback(
```